tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -437,3 +519,54 @@ func verifyNftablesStatus(expectedStatus, nodeName string) {
 	Expect(actualStatus).To(Equal(expectedStatus),
 		fmt.Sprintf("Expected nftables status on %s to be %s, but got %s", nodeName, expectedStatus, actualStatus))
 }
+
+func verifyTCPPortBeforeCustomFirewallActive(
+	masterPod *pod.Builder,
+	testPodWorker0 *pod.Builder,
+	ip4Worker0NodeAddr []string,
+	portNum int,
+) {
+	By("Verify ingress TCP traffic over port 8888 between the master Pod and the test pods on the workers")
+
+	err := cmd.ValidateTCPTraffic(masterPod, ip4Worker0NodeAddr, "net1", frrconfig.ContainerName,
+		portNum)
+	Expect(err).ToNot(HaveOccurred(),
+		"Failed to send ingress TCP traffic over port 8888 to the worker nodes")
+
+	By("Verify egress TCP traffic over port 8888 between the test Pod on the worker0 and the master pod")
+
+	err = cmd.ValidateTCPTraffic(testPodWorker0, []string{"172.16.0.1"},
+		"br-ex", "", portNum)
+	Expect(err).ToNot(HaveOccurred(),
+		"Failed to send egress TCP traffic over port 8888 to the pod on the master node")
+}
+
+func verifyIngressTCPTrafficAfterCustomFirewallActive(
+	masterPod *pod.Builder,
+	testPodWorker0 *pod.Builder,
+	ip4Worker0NodeAddr []string,
+	portNum int,
+) {
+	err := cmd.ValidateTCPTraffic(masterPod, ip4Worker0NodeAddr, "net1", frrconfig.ContainerName,
+		portNum)
+	Expect(err).To(HaveOccurred(),
+		fmt.Sprintf("Successfully sent ingress TCP traffic over port %d to the worker nodes", portNum))
+
+	By(fmt.Sprintf("Verify egress TCP traffic over port %d between the test Pod on the worker0 and the "+
+		"master pod", portNum))
+
+	err = cmd.ValidateTCPTraffic(testPodWorker0, []string{"172.16.0.1"},
+		"br-ex", "", portNum)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to send egress TCP traffic over port %d to the pod on the master node", portNum))
+}
+
+func rebootNodeAndWaitForMcpStable() {
+	_, err := cluster.ExecCmdWithStdout(APIClient,
+		"reboot -f",
+		metav1.ListOptions{LabelSelector: "kubernetes.io/hostname=worker-0"})
+	Expect(err).ToNot(HaveOccurred(), "Failed to reboot worker-0")
```

Comment:
use var instead of hardcoded worker-0 name

Reply:
changed